app/models/metric/capture.rb

```diff
@@ -13,28 +13,27 @@ def self.capture_cols
   end
 
   def self.historical_days
-    (Settings.performance.history.initial_capture_days || 7).to_i
+    Settings.performance.history.initial_capture_days.to_i
   end
 
   def self.historical_start_time
     historical_days.days.ago.utc.beginning_of_day
   end
 
   def self.concurrent_requests(interval_name)
-    requests = Settings.performance.concurrent_requests[interval_name]
-    requests ||= interval_name == 'realtime' ? 20 : 1
-    requests = 20 if requests < 20 && interval_name == 'realtime'
-    requests
+    ::Settings.performance.concurrent_requests[interval_name]
```

Comment:
This is not a default, but an upper bound.  I think it has to stay in the code.  If the user changes it to 1000, we can't allow that value.  Alternately, we could add a line to the Validator to fail if the value is over 20.